cmd/gke-gcloud-auth-plugin/cred.go

```diff
@@ -245,7 +254,8 @@ func (p *plugin) accessToken() (string, *metav1.Time, error) {
 	}
 
 	if useCache {
-		if err := p.writeGcloudAccessTokenToCache(token, *expiry); err != nil {
+		extraArgs := p.tokenProvider.getExtraArgs()
```

Comment:
do extraArgs need to be passed into writeGcloudAccessTokenToCache? Can we retrieve it inside as p.tokenProvider.getExtraArgs()?